sample-authz/src/main/java/net/trajano/ms/example/authz/HazelcastConfiguration.java

```diff
@@ -19,21 +20,35 @@
     @Value("${token.refreshTokenExpiration:3600}")
     private int refreshTokenExpirationInSeconds;
 
+    private static final Logger LOG = LoggerFactory.getLogger(HazelcastConfiguration.class);
+    //
+    //    @Bean
+    //    public HazelcastInstance cacheManager(Config config) {
+    //        return Hazelcast.newHazelcastInstance(config);
+    //    }
+
     @Bean
     public Config hazelcastConfig() {
 
-        return new Config()
+        Config config = new Config()
             .setInstanceName(instanceName)
             .setProperty("hazelcast.logging.type", "slf4j")
             .addMapConfig(new MapConfig()
                 .setName(TokenCache.ACCESS_TOKEN_TO_CLAIMS)
-                .setTimeToLiveSeconds(accessTokenExpirationInSeconds))
+                .setTimeToLiveSeconds(accessTokenExpirationInSeconds)
+                .setMaxIdleSeconds(accessTokenExpirationInSeconds))
             .addMapConfig(new MapConfig()
                 .setName(TokenCache.REFRESH_TOKEN_TO_ACCESS_TOKEN)
-                .setTimeToLiveSeconds(refreshTokenExpirationInSeconds))
+                .setTimeToLiveSeconds(refreshTokenExpirationInSeconds)
+                .setMaxIdleSeconds(refreshTokenExpirationInSeconds))
             .addMapConfig(new MapConfig()
                 .setName(TokenCache.REFRESH_TOKEN_TO_CLAIMS)
-                .setTimeToLiveSeconds(refreshTokenExpirationInSeconds));
+                .setTimeToLiveSeconds(refreshTokenExpirationInSeconds)
+                .setMaxIdleSeconds(refreshTokenExpirationInSeconds));
+        LOG.debug("hazelcast config={}", config);
+        return config;
+        //                .setListenerConfigs(Arrays.asList(new ListenerConfig()
+        //                .setClassName(EntryChangeListener.class.getName())));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)